packages/eslint-config/rules/style.js

```diff
@@ -0,0 +1,775 @@
+module.exports = {
+  rules: {
+    /**
+     * enforce linebreaks after opening and before closing array brackets
+     */
+    'array-bracket-newline': ['off'],
+
+
+    /**
+     * enforce consistent spacing inside array brackets
+     */
+    'array-bracket-spacing': ['error', 'never'],
+
+
+    /**
+     * enforce line breaks after each array element
+     */
+    'array-element-newline': ['off'],
+
+
+    /**
+     * disallow or enforce spaces inside of blocks after opening block and before closing block
+     */
+    'block-spacing': ['error', 'always'],
+
+
+    /**
+     * enforce consistent brace style for blocks
+     */
+    'brace-style': ['error', '1tbs', {
+      allowSingleLine: false,
+    }],
+
+
+    /**
+     * enforce camelcase naming convention
+     */
+    'camelcase': ['error', {
+      properties: 'always',
+      ignoreDestructuring: false,
+      allow: [
+        '^UNSAFE_', // for unsafe React lifecycle methods
+      ],
+    }],
+
+
+    /**
+     * enforce or disallow capitalization of the first letter of a comment
+     */
+    'capitalized-comments': ['off'],
```

Comment:
I'm thinking about turning this on now too.

Reply:
Why? I really think enforcing the grammar of documentation is outside the scope of this, and what if you need to start a comment referring to something that is always written as lowercase.

Reply:
fair nuff :P